newIDE/app/src/GameDashboard/PublicGameProperties.js

```diff
@@ -162,6 +194,35 @@ function PublicGameProperties({
             multiline
             rows={5}
           />
+          {setUserSlug && setGameSlug && (
+            <Line>
```

Comment:
Double check if it looks good on a small (mobile) screen :)

Reply:
Should the drop-down list be disable if there is only one choice?

Reply:
For the game list: we should use a `ResponsiveLineStackLayout` so that on mobile it goes on one column (the thumbnail then the rest)